### PR TITLE
Update complete page for core journey

### PR DESCRIPTION
**Pull request by wilpow-github**
### Context

The /sign-in/complete page is a shared page used by all journeys when inside an OAuth sign in. The content should be different depending on the journey, however. We’re missing the content for the core journey.

### Changes proposed in this pull request

Amend the /sign-in/complete page so it matches the designs in the prototype.

### Checklist

-   [x] Attach to Trello card
-   [ ] Rebased master
-   [ ] Cleaned commit history
-   [ ] Tested by running locally
-   [ ] Reminder created to manually clean any removed app settings post deployment

### CODE REVIEW

File: dotnet-authserver/tests/TeacherIdentity.AuthServer.EndToEndTests/Register.cs [hunk 1]
```diff
@@ -68,6 +71,93 @@ public async Task NewUserCanRegister()
             });
     }
 
+    [Fact]
+    public async Task NewUserWithTrnRequiredCanRegister()
```
Comment (wilpow-github) [inline dotnet-authserver/tests/TeacherIdentity.AuthServer.EndToEndTests/Register.cs:75]: Ticket mentioned multiple e2e tests. I've added this one, is there something I'm missing?

Comment (gunndabad) [reply dotnet-authserver/tests/TeacherIdentity.AuthServer.EndToEndTests/Register.cs:75]: Would be good to cover existing email and existing mobile phone scenarios too.

File: dotnet-authserver/tests/TeacherIdentity.AuthServer.EndToEndTests/PageExtensions.cs [hunk 2]
```diff
@@ -309,15 +315,93 @@ public static async Task SubmitCompletePageForNewUser(this IPage page)
...
+    public static async Task SubmitExistingAccountPhonePage(this IPage page)
+    {
+        await page.WaitForUrlPathAsync("/sign-in/register/existing-account-phone");
+        await page.ClickButton("Request security code");
+    }
+
+    public static async Task SubmitExistingAccountPhoneConfirmationPage(this IPage page)
+    {
+        await page.WaitForUrlPathAsync("/sign-in/register/existing-account-phone-confirmation");
+        await page.FillAsync("label:text-is('Security code')", HostFixture.UserVerificationPin);
```
Comment (wilpow-github) [inline dotnet-authserver/tests/TeacherIdentity.AuthServer.EndToEndTests/PageExtensions.cs:392]: Called Security code here and Confirmation code for Existing Email Confirmation

File: dotnet-authserver/src/TeacherIdentity.AuthServer/AuthenticationState.cs [hunk 3]
```diff
@@ -358,6 +358,7 @@ public void OnMobileNumberVerified(User? user = null)
         }
 
         MobileNumberVerified = true;
+        FirstTimeSignInForEmail = user is null;
```
Comment (wilpow-github) [inline dotnet-authserver/src/TeacherIdentity.AuthServer/AuthenticationState.cs:361]: Sign in via exisiting phone found should not show first time sign in complete message

**Review by gunndabad — APPROVED** [review]

**Pull request merged**